kpatch-build/kpatch-build

```diff
@@ -963,6 +963,8 @@ ld -r $KPATCH_LDFLAGS -o ../patch/tmp_output.o $(find . -name "*.o") 2>&1 | logg
 
 if [[ "$USE_KLP" -eq 1 ]]; then
 	cp "$TEMPDIR"/patch/tmp_output.o "$TEMPDIR"/patch/output.o || die
+	# Avoid MODPOST warning and error (v5.8+) with an empty .cmd file
+	touch "$TEMPDIR"/patch/.output.o.cmd
```

Comment:
Thanks for working on it. A minor nitpick, should `|| die` be used like other commands to fail the build based on the return code after its run? One argument would be that if the previous command `cp` succeeds in creating `output.o`, theoretically creating `.output.o.cmd` should also succeed.  On other hand if the die is not used and if the touch commands fails to create the `.output.o.cmd` file, the build will fail at every next step of kernel module build.  I am little biased towards including `die` here.